StudentList/Common/Extensions/StudentFilterExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using StudentList.Models;
+
+namespace StudentList.Common.Extensions
+{
+    public static class StudentFilterExtensions
+    {
+        public static bool IsDefault(this StudentFilter studentFilter)
+        {
+            if (studentFilter.Name == StudentFilter.Default.Name
+                && studentFilter.Group == StudentFilter.Default.Group
+                && studentFilter.Birthdate == StudentFilter.Default.Birthdate)
+            {
+                return true;
```

Comment:
can be written as `return studentFilter.Name == StudentFilter.Default.Name
                && studentFilter.Group == StudentFilter.Default.Group
                && studentFilter.Birthdate == StudentFilter.Default.Birthdate`